packages/@magic-sdk/provider/src/core/view-controller.ts

```diff
@@ -88,6 +90,7 @@ async function persistMagicEventRefreshToken(event: MagicMessageEvent) {
 }
 
 export abstract class ViewController {
+  private isReady = false;
```

Comment:
nit: The naming convention here is little confusing with `this.ready` already here. Is there a better name to describe what this is actually checking for?

Reply:
I like it, go for it!